webapp/tests/forms/flows/test_eligibility_step_chooser.py

```diff
@@ -75,3 +76,70 @@ def test_set_attributes_correctly(self):
             self.assertEqual(self.testing_steps, list(step_chooser.steps.values()))
             self.assertEqual(None, step_chooser.overview_step)
 
+
+class TestEligibilityStepChooserDeterminePrevStep(unittest.TestCase):
+    def setUp(self):
+        self.step_chooser = EligibilityStepChooser('eligibility')
+        self.step_chooser.step_order = []
+        self.step_chooser.steps = {}
+
+    def set_up_steps(self, steps_definitions):
+        for step_definition in steps_definitions:
+            step_name = step_definition['name']
+            mocked_step = MagicMock(is_previous_step=MagicMock(return_value=step_definition['is_previous_step']))
+            mocked_step.name = step_name
+            self.step_chooser.step_order.append(step_name)
+            self.step_chooser.steps[step_name] = mocked_step
+
+    def test_if_step_is_first_step_then_return_none(self):
```

Comment:
While looking at it, I am wondering if we should add another test for what happens if it is the first step and another step returns true

Reply:
Well, they shouldn't be called at all - which we test in another test - so they can't be returned?

Reply:
To make it more clear: set one False to True

Reply:
Change order: standard behaviour first, then rest